miscelanea/004-route-traffic/poc-004-2.yaml

```diff
@@ -0,0 +1,204 @@
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "80"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 80
+          hostPort: 80
+          protocol: TCP
+    - name: https
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "443"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 443
+          hostPort: 443
+          protocol: TCP
+    - name: ldap
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "389"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 389
+          hostPort: 389
+          protocol: TCP
+    - name: ldaps
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "636"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 636
+          hostPort: 636
+          protocol: TCP
+    - name: kerberos-tcp
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "88"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 88
+          hostPort: 88
+          protocol: TCP
+    - name: kerberos-admin-tcp
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "464"
+        - name: "ECHO_PROTOCOL"
+          value: "tcp"
+      ports:
+        - containerPort: 464
+          hostPort: 464
+          protocol: TCP
+    - name: kerberos-udp
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "88"
+        - name: "ECHO_PROTOCOL"
+          value: "udp"
+      ports:
+        - containerPort: 88
+          hostPort: 88
+          protocol: UDP
+    - name: kerberos-admin-udp
+      image: quay.io/freeipa/freeipa-operator:poc-004
+      securityContext:
+        capabilities:
+          add:
+            - NET_BIND_SERVICE
+      env:
+        - name: "ECHO_PORT"
+          value: "464"
+        - name: "ECHO_PROTOCOL"
+          value: "udp"
+      ports:
+        - containerPort: 464
+          hostPort: 464
+          protocol: UDP
+---
+apiVersion: v1
```

Comment:
It is required to define a Service object for making possible to communicate services between two different pods?

Reply:
Yes it is.